OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -199,13 +218,19 @@ public virtual void Activate(Actor self, Order order, SupportPowerManager manage
 
 		public virtual void PlayLaunchSounds()
 		{
-			var renderPlayer = Self.World.RenderPlayer;
-			var isAllied = Self.Owner.IsAlliedWith(renderPlayer);
+			var localPlayer = Self.World.LocalPlayer;
+
+			if (localPlayer == null || localPlayer.Spectating)
+				return;
+
+			var isAllied = Self.Owner.IsAlliedWith(localPlayer);
 			Game.Sound.Play(SoundType.UI, isAllied ? Info.LaunchSound : Info.IncomingSound);
```

Comment:
This smells wrong - I would expect that the `LaunchSound` should be a world-based sound that works like the weapon firing sound and plays for everyone (and that `IncomingSound` didn't exist at all).

This weirdness already existed so any fix should be left to a followup PR, just raising it here for awareness.

Reply:
Agree - I found it odd too. It seems to be a remnant of an earlier implementation of launch notifications :thinking: In any case `IncomingSound` is not used anywhere and `LaunchSound` is used only in one place in TS.